pages/mesosphere/dcos/services/beta-kafka/2.10.0-2.4.0-beta/release-notes/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Release notes for 2.10.0-2.4.0-beta
+title: Release Notes
+menuWeight: 5
+model: /mesosphere/dcos/services/kafka/data.yml
+render: mustache
+---
+
+## Version 2.10.0-2.4.0-beta
+
+### Updates
+- Added custom TLS certificate support for Kafka.
```

Comment:
When the TLS Cert sections have been moved to `security/index.md` link them here.